1904-12-12.xml

```diff
@@ -0,0 +1,6157 @@
+                        <p>The S.S. Scottish Prince arrived from Antwerp and London with passengers
+                            and general cargo, etc., on Sunday. </p>
+                        <p>The Messageries Maritimes S.S. Salazie, from Marseilles, passed the
+                            Straits of Messina on Saturday and is due here to-morrow morning. </p>
+                    </div>
+                    <div type="item">
+                        <head> Calendar of Coming Events. </head>
+                        <p> ALEXANDRIA: </p>
+                        <p> December, </p>
+                        <p>Mon. 12 National Bank of Egypt, Annual General Meeting, Anglo-Egyptian
+                            Spinning and Weaving Company, 8 p.m. Zizinis Theatre I Puritans. 9pm.
+                            Alhamhra Theatre, Ballet and Variety Company. 9 p.m. </p>
+                        <p>Fri. 16 St. Andrew's School, Missale, G of S. Women's Guild Sale of Work.
+                            Alderson's Garden. Variety Entertainment, 4p.m. </p>
+                        <p>Sat. 17 A. L. M. D. 8. "A Great Philanthropic Work in London," lecture by
+                            Mr. Atwood. 9 p.m." </p>
+                        <p> CAIRO. </p>
+                        <p> December.</p>
+                        <p> Mon. 18 Verdi Theatre, Italian opera company 9 p.m, Theatre des
+                            Nouveautes. Variety entertainment. 9.30 p.m. Trocadero. Variety
+                            Entertainment, 9.30 p.m. </p>
+                        <p>Fri. 16 Mens House. Gymkhana. Y. W.C. A. Rooms. Sale of Work, to be
+                            opened by Lady Cromer. 8.30 p.m. </p>
+                        <p>Sat, 17 Savoy Hotel Small Dance. 10 p.m. Shepheard's Hotel. Band Royal
+                            Inniskilling Fusiliers 4 to 6 pm.</p>
+                        <p> Sun. 18 Cerole Artistique. Beauty Show. 8 p.m, Zoological Gardens."
+                            Performances by Band of H.A. Infantry in afternoon </p>
+                        <p>Wed. 21 K. S. C. grounds. First Winter Races, 2 p.m. Mens House Hotel.
+                            Small Dance, 10 p.m. </p>
+                        <p>Fri. 28 K. S. C. grounds. First Winter Races, 2nd day, p.m. </p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-cah02">
+                        <head>CARLTON HOTEL</head>
+                        <p>Bulkeley (near Alexandria.)</p>
+                        <p>Ramleh's Fashionable Hotel.</p>
+                        <p>Patronized by the Elite.</p>
+                        <p>Full Pension P.T. 50 a day. Visitors fron Cairo alight at Sidi-Gaber
+                            Station.</p>
+                        <p><measure type="indexNo">21440-24-5-905</measure></p>
+                        <p>G. AQUILINA, Proprieter.</p>
+                    </div>
+                </div>
+                <pb n="4"/>
```

Comment:
You need an extra `</div>` here to close page 3. At present page 4 is _inside_ page 3.